src/main/scala/nl.knaw.dans.easy.springfield/Command.scala

```diff
@@ -72,7 +73,7 @@ object Command extends App
         actions <- createSpringfieldActions(videos)
       } yield (new PrettyPrinter(160, 2).format(actions), parentsToCreate)
       result.map { case (s, ps) =>
-        println(s)
+        println(addPrologue(s))
```

Comment:
Something like this?
`val writer = new StringWriter()
        XML.write(writer, XML.loadString(s), "UTF-8", xmlDecl = true, null)
        println(writer.toString)`

Reply:
what about:

```scala
// top of the file
import resource.managed
import java.io.OutputStreamWriter
import java.nio.charset.StandardCharsets
import scala.xml.XML

// instead of println(s)
managed(new OutputStreamWriter(Console.out))
  .acquireAndGet(XML.write(_, XML.loadString(s), StandardCharsets.UTF_8.name, xmlDecl = true, null))
```

or even better, move the `XML.loadString` up to wrap the `PrettyPrinter.format`

Reply:
It works, i don't know if i understood the moving up correctly though 😅 


`yield (
        managed(new OutputStreamWriter(Console.out))
          .acquireAndGet(XML.write(_, XML.loadString(new PrettyPrinter(160, 2).format(actions)), StandardCharsets.UTF_8.name, xmlDecl = true, null)), parentsToCreate
      )`